### DISCUSSION TITLE
Youtube dl for cpanel

### DISCUSSION THREAD

**Discussion by lilzychuk** [norkunas/youtube-dl-php#228] [General]
hello, I have a php website which is music streaming platform like Spotify that's hosted on cPanel. The website play a music through youtube and i want it to be able to download music from youtube. Unfortunately, i am unable to install youtube-dl on file, can you please help me. my website is https://viontic.com
![Opera Snapshot_2024-01-25_205948_viontic com](https://github.com/norkunas/youtube-dl-php/assets/130857752/a2669b98-0370-4d42-a544-5f077f0a9deb)

**Comment by norkunas** [comment]
Hi, I think cpanel has nothing to do it, if you are using shared host and they don't allow to install external binaries then there are no options
